sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ParameterNamesInPartialMethod.CSharp9.cs

```diff
@@ -0,0 +1,19 @@
+﻿public partial record Record
+{
+    partial void M1(int x, int y);
+
+    private partial void M2(int x, int y);
+    internal partial void M3(int x, int y);
+    protected partial void M4(int x, int y);
+    protected internal partial void M5(int x, int y);
+    public partial void M6(int x, int y);
```

Comment:
to test the [extended support](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/extending-partial-methods) for partial methods, you could also add a non-void method like
```
partial bool TryFoo(string input, out string output);
```